docs/REST API/rest-api.md

```diff
@@ -9824,9 +9824,10 @@ Returns information about the specified Fleet-maintained app.
 {
   "fleet_maintained_app": {
     "id": 1,
+    "url": "https://downloads.1password.com/mac/1Password-8.10.50-aarch64.zip",
     "name": "1Password",
-    "filename": "1Password-8.10.44-aarch64.zip",
-    "version": "8.10.40",
+    "filename": "1Password-8.10.50-aarch64.zip",
```

Comment:
Since this is attached to API changes of a story, forwarding that filename is unused and is being removed: https://github.com/fleetdm/fleet/pull/25685